frontend/src/components/client/models/getClientFeature.tsx

```diff
@@ -33,6 +33,7 @@ const clientListConfig = (config: ClientFeatureBuilderConfig): IList<ClientModel
         <span>{client.btw}</span>
       </>
     ),
+    sort: (asc) => (c1, c2) => sortResult(c1.name.localeCompare(c2.name) > 0, asc)
```

Comment:
Ik zou zelfs iets verder gegaan zijn en hier gezet hebben `sort: true`

En dan logic ala:
```
if (sort === true) {
   // sort on element[key]
  // if (typeof element[key] === 'string') --> .localeCompare sort
}
```

Maar laat dit maar zo, expliciet is ook OK.

Het voordeel van de `sort: true` zou vb zijn dat je op een bepaald moment kan kiezen om de .localeCompare door iets  complexer te vervangen (vb: met een latinize erbij)